### PR TITLE
Add milkTIA token to Osmosis's asset list 

### PR DESCRIPTION
**Pull request by josietyleung**

### CODE REVIEW

File: osmosis/assetlist.json [hunk 1]
```diff
@@ -1023,6 +1023,31 @@
...
+          "exponent": 0
+        },
+        {
+          "denom": "milktia",
+          "exponent": 6
+        }
+      ],
+      "base": "factory/osmo1f5vfcph2dvfeqcqkhetwv75fda69z7e5c2dldm3kvgj23crkv6wqcn47a0/umilktia",
+      "name": "milkTIA",
+      "display": "milkTIA",
```
Comment (JeremyParish69) [inline osmosis/assetlist.json:1041]: 'milkTIA' is not defined under denom_units. (looks like this should be 'milktia')

Comment (josietyleung) [reply osmosis/assetlist.json:1041]: Hey @JeremyParish69 updating the denom_units to be milkTIA  to align

**Comment by JeremyParish69** [comment]
Can I ask why the symbol for this has upper and lower case letters? We'd like to have all token symbols use only uppercase unless they are a derivative of another token. (Is this a liquid staked $TIA perhaps?)

**Comment by josietyleung** [comment]
> Can I ask why the symbol for this has upper and lower case letters? We'd like to have all token symbols use only uppercase unless they are a derivative of another token. (Is this a liquid staked $TIA perhaps?)

Hey yup this is liquid staked $TIA indeed!

**Comment by JeremyParish69** [comment]
Ok, we should add a trace to TIA, in that case. A rather simple change I don't mind taking care of. I just need to know the provider.

**Comment by josietyleung** [comment]
> Ok, we should add a trace to TIA, in that case. A rather simple change I don't mind taking care of. I just need to know the provider.

See that you've added it already. Thank you!

**Pull request merged**